js/build.js

```diff
@@ -101,10 +103,10 @@ Fliplet.Widget.instance('image-gallery', function (data) {
 
   // Appearance change Hook
   Fliplet.Hooks.on('appearanceChanged', function () {
-    initGallery();
+    initGallery({ appendImages: false});
```

Comment:
@upplabs-alex-levchenko @YaroslavOvdii Here's an error that should not be picked up by us.

A single-line object should have spaces inside the `{}` braces.

```suggestion
    initGallery({ appendImages: false });
```

Reply:
@MaksymShokin We don't need to provide a parameter here if the default behaviour is not to append images.

Reply:
Fixed